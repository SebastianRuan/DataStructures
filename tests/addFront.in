a 5 0
a 12 0
a 333 0
a 99 0
