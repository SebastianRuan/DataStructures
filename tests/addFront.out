f d:333 i:1
f d:99 i:0
f d:0 i:-1
99
333
12
5
